admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,534 @@
+system:build-strategy-source-binding            /system:build-strategy-source                              system:authenticated
+system:replication-controller                   /system:replication-controller                                                                            openshift-infra/replication-controller
+system:sdn-readers                              /system:sdn-reader                                         system:nodes
+system:build-strategy-docker-binding            /system:build-strategy-docker                              system:authenticated
+system:routers                                  /system:router                                             system:routers
+system:oauth-token-deleters                     /system:oauth-token-deleter                                system:authenticated, system:unauthenticated
+system:node-proxiers                            /system:node-proxier                                       system:nodes
+system:nodes                                    /system:node                                               system:nodes
+self-provisioners                               /self-provisioner                                          system:authenticated:oauth
+system:service-serving-cert-controller          /system:service-serving-cert-controller                                                                   openshift-infra/service-serving-cert-controller
+system:registrys                                /system:registry                                           system:registries
+system:pv-binder-controller                     /system:pv-binder-controller                                                                              openshift-infra/pv-binder-controller
+system:build-strategy-jenkinspipeline-binding   /system:build-strategy-jenkinspipeline                     system:authenticated
+system:deployment-controller                    /system:deployment-controller                                                                             openshift-infra/deployment-controller
+system:masters                                  /system:master                                             system:masters
+system:service-load-balancer-controller         /system:service-load-balancer-controller                                                                  openshift-infra/service-load-balancer-controller
+----
+====
+
+These commands can generate huge lists, so you may want to pipe the output into a text file that you can search through more easily.
+
+[[control-user-roles]]
+== Controlling User Permissions with Roles
+
+You may want to define
+link:../admin_guide/manage_authorization_policy.html[roles] (or permissions) for a user, even before they ever logged in. That way when they first log in, their roles are already configured and they can start working immediately. You can assign many different types of roles to users such as admin, basic-user, self-provisioner, and cluster-reader.
+
```

Comment:
Not sure if the tense is right in the first sentence. Maybe:

"You can define roles (or permissions) for a user before their initial log in so they can start working immediately."